common/djangoapps/microsite_configuration/admin.py

```diff
@@ -15,4 +15,24 @@ class Meta(object):  # pylint: disable=missing-docstring
         model = Microsite
 
 
+class MicrositeHistoryAdmin(admin.ModelAdmin):
```

Comment:
OK, let me merge this as it stands and then I'll make a new branch and try to convert the models over